src/app/scripts/cac/map/cac-map-templates.js

```diff
@@ -232,6 +232,7 @@ CAC.Map.Templates = (function (Handlebars, moment, Utils) {
                     '</div>',
                 '</div>',
             '</div>',
+            '<div class="route-summary-go-btn">Go</div>',
```

Comment:
It's visible in mobile only. Client feedback indicated it's not entirely clear you have to tap the itinerary item at bottom of screen to open it. This is a dummy button, of sorts, to make that clear.